google-cloud-vision/README.md

```diff
@@ -1,59 +1,42 @@
-# google-cloud-vision
+o# Ruby Client for Google Cloud Vision API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
```

Comment:
What is the first character in this line? (`o`) Should it be there?

Reply:
Whoops, typo. Removed.